hnn_core/optimization.py

```diff
@@ -320,16 +365,82 @@ def _run_optimization(maxiter, param_ranges, optrun):
     return result
 
 
-def optimize_evoked(params, target_dpl, initial_dpl, maxiter=50,
+def _get_drive_params(net, drive_names):
```

Comment:
This might be useful in the future when we think about more easily exposing the drives in the network